qiskit/algorithms/amplitude_estimators/mlae.py

```diff
@@ -240,7 +240,7 @@ def compute_confidence_interval(
             AlgorithmError: If `run()` hasn't been called yet.
             NotImplementedError: If the method `kind` is not supported.
         """
-        interval = None
+        interval: typing.Tuple[float, float] | None = None
```

Comment:
Needs to be `tuple[float, float] | None`. In general, I usually don't see `typing.X` pattern ever being used.